### PR TITLE
HCLANG-34 Added/ changed introductions

### PR DESCRIPTION
**Pull request by nillas12**
### Description
Added/ changed the introduction parts to all already existing sections

### Changelog
-Changed introduction sections
-Removed all mentions of HCL in the text, as it wasn't mentioned yet.

### Issues solved 
add these with the respective key as HCLANG-1 (as an example)
 - HCLANG-34

### Issues opened as a result of this 
Add something here if new issues arise because of this. For instance if new features-requests or bugs arise from this PR. If this is not the case, then remove this section.

## FOR REVIEWER [Do not remove]
A small checklist of things to note when you are doing a review. Make sure these things apply before going into master.
 - Did the PR actually fix the issue
 - Comment anything that doesn't make sense or anything you want clarified. Many comments are always ok
 - Make sure this is ready for exams if this is going into master
 - It's always a good idea having another reviewer look at a PR as well, if it's major. get a buddy.

### Regarding filenames
- all .tex files use pascal case
- all other files [except in /code/] use snake case

### Regarding code [if applicable]
- Check [coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
- Are public methods documented? 
- Are there testcases for new features / methods?
- Have you tried running the source and tested things that cannot be test-cased?

### Regarding Report [if applicable]
- Does each line have a linechange in the sourcecode?
- Does sentences make sense?
- Did you do spellchecking?
- Citations?

### CODE REVIEW

File: report/2.Analysis/ArduinoLanguagesCtrl.tex [hunk 1]
```diff
@@ -1,5 +1,5 @@
 \section{Existing languages}
-Before doing any work on HCL, the project group has decided to research how many high-level languages already exist for the Arduino platform, and what distinguishes them from each other.
+Before doing any work on the groups own language,it was decided to look into the high-level languages that already exist for the Arduino platform, and what distinguishes them from each other.
```
Comment (Abildgaard96) [inline report/2.Analysis/ArduinoLanguagesCtrl.tex]: groups = plural
group's = ejefald

Comment (Abildgaard96) [inline report/2.Analysis/ArduinoLanguagesCtrl.tex]: add space between "language,it"

File: report/2.Analysis/LanguagesForYoungPeople.tex [hunk 2]
```diff
@@ -2,7 +2,7 @@ \section{Languages for beginners}
 
 There are a wide variety of different tools, games and languages with the intend to teach students how to program. 
 These span from very specific game creation tools, to general purpose programming languages. 
-In order to gather some inspiration for the HCL programming language some of the existing solutions, and most common languages for beginners have been analysed.
+In order to gather some inspiration for the groups own programming language, some of the existing solutions, and most common languages for beginners have been analysed.
```
Comment (Abildgaard96) [inline report/2.Analysis/LanguagesForYoungPeople.tex]: group's! not groups

Comment (nillas12) [reply report/2.Analysis/LanguagesForYoungPeople.tex]: Changed, fair point

File: report/2.Analysis/ArduinoLanguagesCtrl.tex (hunk 1 above)
Comment (nillas12) [reply report/2.Analysis/ArduinoLanguagesCtrl.tex]: Has been changed

Comment (nillas12) [reply report/2.Analysis/ArduinoLanguagesCtrl.tex]: Has been changed

File: report/2.Analysis/LanguagesForYoungPeople.tex (hunk 2 above)
Comment (nillas12) [reply report/2.Analysis/LanguagesForYoungPeople.tex]: Changed, fair point

**Pull request merged**